contrib/vdirsyncer.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <!-- click requires a proper UTF-8 locale -->
+    <key>EnvironmentVariables</key>
+    <dict>
+           <key>LANG</key>
+           <string>en_US.UTF-8</string>
+           <key>LC_ALL</key>
+           <string>en_US.UTF-8</string>
```

Comment:
The people over @homebrew suggest the best way would be to have some placeholder values like @@PREFIX@@ for such values. rationale is that people either use some sort of package manager, which can replace those values with the actual ones, or install manually where they need to provide them anyway. not really sure how this would work e.g. with pip.
but for homebrew this would be the best solution and give vdirsyncer user a blueprint in case they install manually. this would also alleviate your concern regarding hard-coded values. would that be ok?